libs/mini-rx-store/src/lib/store-core.ts

```diff
@@ -54,11 +56,13 @@ class StoreCore {
     private extensions: StoreExtension[] = [];
 
     constructor() {
+        this.combineReducersFn = defaultCombineReducers;
```

Comment:
Super minor, but I would rather initialise the `private combineReducersFn` at declaration on line 37.

`private combineReducersFn: CombineReducersFn<AppState> = defaultCombineReducers;`

A little bit more consistent with the other props.

Reply:
yes, we generally tried to be consistent with the style.